frontend/public/lib/locationOperations.js

```diff
@@ -214,3 +217,34 @@ export function getLocationValue(values, locationKey) {
   }
   return values[locationKey];
 }
+
+//When filtering by location, the url only holds the place_id, osm_id and loc_type, but not the name
+//This function is used to retrieve the whole location object
```

Comment:
Nit: just comment formatting on func declaration vs. `//` 
```suggestion
/** 
 * When filtering by location, the url only holds the place_id, osm_id and loc_type, but not the name. This is used to retrieve the whole location object
*/
```